src/react/src/components/Contribute/ProductionLocationDialog.jsx

```diff
@@ -72,9 +75,20 @@ const getStatusBadgeClass = (classes, status) => {
 };
 
 const getPendingTooltipText = claimStatus => {
-    if (claimStatus === PRODUCTION_LOCATION_CLAIM_STATUSES_ENUM.UNCLAIMED) {
+    const {
+        PENDING,
+        CLAIMED,
+        UNCLAIMED,
+    } = PRODUCTION_LOCATION_CLAIM_STATUSES_ENUM;
+
+    if (claimStatus === PENDING || claimStatus === CLAIMED) {
+        return 'Your submission is under review. You will receive a notification once the production location is live on OS Hub.';
+    }
+
+    if (claimStatus === UNCLAIMED) {
```

Comment:
These changes conflict with [OSDEV-1831](https://opensupplyhub.atlassian.net/browse/OSDEV-1831) that were implemented earlier.

Reply:
@mazursasha1990 This PR implements the fixes originally suggested in this [comment](https://opensupplyhub.atlassian.net/browse/OSDEV-1787?focusedCommentId=14743).

Reply:
As per our discussion, I left only tests in this PR.